### ISSUE TITLE
Upcoming release

### ISSUE THREAD

**Issue by nandorojo** [nandorojo/moti#137]
The upcoming release of Moti will address a few things:

1. Simplifying imports
2. Reanimated 2.3 support
3. Expo SDK 44 support

## Simplifying imports

For 1, which is currently in progress at #136, there will no longer be separate packages for `@motify/skeleton` and `@motify/interactions`.

Instead, you will only need to install `moti`, and you will import these packages from `moti/skeleton` and `moti/interactions`. This makes for a much nicer developer experience, without sacrificing our current tree shaking. You can read more about this at the PR  (#136).

### Testing

```sh
yarn remove @motify/interactions @motify/skeleton
yarn add moti@canary
```

And then replace all your imports to be `moti/interactions` and `moti/skeleton`.

## Reanimated 2.3 support

Since upgrading to Reanimated 2.3, I've noticed some general regressions in our app. Namely, some timing animations don't work. There are also breaking type issues (#129) and issues with Moti sequences (#131) which may be due to breaking changes.

The next version of Moti will address these changes as well.

**Comment by nandorojo** [comment]
It would be great to get help testing this.

You should be able to do this now:

```sh
yarn remove @motify/skeleton @motify/interactions
yarn add moti@import-experiment
```

And then find and replace all imports on VSCode:
`@motify/interactions` -> `moti/interactions`
`@motify/skeleton`       -> `moti/skeleton`

**Comment by axeldelafosse** [comment]
Just tested, it's working ✨ Thanks!

**Comment by nandorojo** [comment]
Released in `0.17.0`

```sh
yarn add moti
```

Notes: https://github.com/nandorojo/moti/releases/tag/v0.17.0
